### PR TITLE
Final changes to reviews module frontend

### PR DESCRIPTION
**Pull request by eric1052**

### CODE REVIEW

File: client/src/components/Pagination.jsx [hunk 1]
```diff
@@ -2,44 +2,32 @@ import React from 'react';
...
     <div className={styles.wrapper}>
-      <ul>
-        {pages.map((page) => (
-          <li key={page} className={styles.pagination}>
-            <input type="button" className={`pageButton-${page} ${styles.pageButton}`} onClick={(e) => onClickHandler(e)} value={page} />
-          </li>
-        ))}
-      </ul>
+      {pages.map((page) => (
+        page === Number(currentPage)
```
Comment (eLeiLei) [inline client/src/components/Pagination.jsx:19]: Is currentPage converted to something other than a Number at some point?

File: client/src/components/ReviewItem.css [hunk 2]
```diff
@@ -64,3 +70,8 @@
 .readMore {
   color: #00A699;
 }
+
+.divider {
+  border: 0.5px solid rgb(230, 230, 230);
+  margin: 2em 0px;
+}
```
Comment (eLeiLei) [inline client/src/components/ReviewItem.css:77]: newline

File: client/src/components/ReviewItem.css [hunk 3]
```diff
@@ -64,3 +70,8 @@
 .readMore {
   color: #00A699;
 }
+
+.divider {
+  border: 0.5px solid rgb(230, 230, 230);
+  margin: 2em 0px;
```
Comment (eLeiLei) [inline client/src/components/ReviewItem.css:76]: Could use 0em purely for consistency

File: client/src/components/ReviewItem.jsx [hunk 4]
```diff
@@ -35,22 +35,25 @@ class ReviewItem extends Component {
...
-          <span>{this.formatDate()}</span>
-        </div>
-        <div className={styles.desc}>
-          {shortened ? (
-            <div>
-              <span>{this.getShortenedDesc()}</span>
-              <span className={styles.readMore} onClick={(e) => this.onClickHandler(e)}>Read More</span>
-            </div>
-          ) : <span>{body}</span> }
+      <div className="wrapper">
```
Comment (eLeiLei) [inline client/src/components/ReviewItem.jsx:38]: This class isn't using the styles class that you've imported.

File: client/src/components/ReviewItem.css (hunk 3 above)
Comment (eric1052) [reply client/src/components/ReviewItem.css:76]: Good point!

File: client/src/components/ReviewItem.jsx (hunk 4 above)
Comment (eric1052) [reply client/src/components/ReviewItem.jsx:38]: Good catch—will change this immediately.

File: client/src/components/Pagination.jsx (hunk 1 above)
Comment (eric1052) [reply client/src/components/Pagination.jsx:19]: This was confusing for me too. I initially had 'page === currentPage' but something kept breaking, and only during the debugging process I realized that e.target.value (which is what I ultimately assign to currentPage) returns a string by default.

**Pull request merged**